common4j/src/main/com/microsoft/identity/common/java/nativeauth/controllers/results/INativeAuthCommandResult.kt

```diff
@@ -72,4 +72,15 @@ interface INativeAuthCommandResult {
         open val correlationId: String = DiagnosticContext.INSTANCE.threadCorrelationId,
         open val errorCodes: List<Int>? = null
     )
+
```

Comment:
If we're checking for blank usernames in the SDK interface, then we can remove the ` ArgUtils.validateNonNullArg(username, "username")` from the respective request classes right?

Reply:
Done